Makefile

```diff
@@ -310,7 +310,7 @@ operator-sdk:
 bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
 	GOFLAGS="-mod=mod" $(OPERATOR_SDK) generate kustomize manifests -q
 	cd config/manager && GOFLAGS="-mod=mod" $(KUSTOMIZE) edit set image controller=$(IMG)
-	GOFLAGS="-mod=mod" $(KUSTOMIZE) build config/manifests | GOFLAGS="-mod=mod" $(OPERATOR_SDK) generate bundle -q --extra-service-accounts "velero" --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+	GOFLAGS="-mod=mod" $(KUSTOMIZE) build config/manifests | GOFLAGS="-mod=mod" $(OPERATOR_SDK) generate bundle -q --extra-service-accounts "velero,openshift-adp-non-admin-controller-manager" --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
```

Comment:
Please help me understand Why do we need this extra service account ? I am not sure if we need changes to the Makefile here. Maybe I am missing something ?

Reply:
this is the NonAdmin ServiceAccount (same one from non admin repo) that will be applied to non admin deployment (and only that)

OADP also ship OADP (applied to oadp controller manager) and Velero (applied to velero deployment) ServiceAccounts

we could merge both (this and OADP's) but I think is safer (do not add unnecessary permissions to non admin deployment) and more organized to split them. Also, this makes update script way more easy to write 

are this points enough to us to ship a new ServiceAccount?

Reply:
Since we already split, I am ok with one more SA to be really clear in roles/rolebindings what is required to add NAC.